src/MensaWidget/MWCanteen.class.st

```diff
@@ -33,20 +33,76 @@ Class {
 	#'squeak_changestamp' : 'Louis Christ 5/31/2024 14:42'
 }
 
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 00:26'
+}
+MWCanteen class >> canteenMorphBorder [
+
+	^ 5
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 12:44'
+}
+MWCanteen class >> canteenMorphColor [ 
+
+	^ Color fern lighter
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:26'
+}
+MWCanteen class >> canteenMorphWidth [
+	
+	^ 410
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:57'
+}
+MWCanteen class >> cellGap [
```

Comment:
Shouldn't it be "canteenMorphCellGab" for consistency? However, we can still adjust the names and consider whether we want to remove "canteenMorph" altogether.